rfcx/arbimon
# Feature/31 Export graph as image

## ✅ DoD

- [x] Resolves #31 

## 📝 Summary

- Add ability to export chart into png image
- Move chart title into chart component and add download button next to it

## 💉 Testing

1. Select a filter to where the data exist (01-07/04/2021) -- download button should be shown up
2. Click on the download button -- file will be auto download to your computer directly

## 💡 More ideas

#73  
- improve filename to include project name 
- add indicator to show which color stands for which group of data in chart that has more than 2 comparisons

#74 
- improve the overall style of each graph section

## 📸 Screenshots


https://user-images.githubusercontent.com/9149523/135063363-6b921c7b-60f4-44c7-9b3b-3cfcd7fe514d.mov
